WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/DashboardDomainRegistrationCardCell.swift

```diff
@@ -0,0 +1,110 @@
+import UIKit
+import WordPressFlux
+
+final class DashboardDomainRegistrationCardCell: BaseDashboardDomainsCardCell {
+
+    // MARK: - View Model
+
+    override var viewModel: DashboardDomainsCardViewModel {
+        return cardViewModel
+    }
```

Comment:
Can we not write this as?

```Swift
override var viewModel: DashboardDomainsCardViewModel = { ... }()
```

Or do we need the "lazy" for functionality?

Reply:
@alpavanoglu If you check the `BaseDashboardDomainsCardCell` class, the `viewModel` setter is private, so I can't override it.

I think it was implemented this way so the `viewModel` property can only be mutated by subclasses. The author of `BaseDashboardDomainsCardCell` might have more context about this. ( cc @staskus ).

Reply:
@alpavanoglu I also want to mention that this approach is inspired by other `BaseDashboardDomainsCardCell ` subclasses. See `DashboardDomainsCardCell` for example.

Reply:
Nice, happy to see the base cell class reusability being useful! 

> I think it was implemented this way so the viewModel property can only be mutated by subclasses.

Yes, exactly! Although I think there's more than one way to implement and structure this. So, if you find any more preferable way feel free to make improvements or changes both in base and child classes 👍